packages/kotti-navbar/src/KtNavbarLogo.vue

```diff
@@ -1,17 +1,36 @@
 <template>
-	<div :style="{ color: themeColor.textColor }">
-		<div :class="objectClass" @click="$emit('logoClick')">
-			<img
-				:src="themeColor.logoUrl"
-				v-if="themeColor.logoUrl && !isNarrow"
-				:height="themeColor.logoHeight"
+	<div :style="{ color: themeColor.textColor, height: '100%' }">
+		<div v-if="isNarrow" :class="objectClass" @click="$emit('logoClick')">
+			<i
+				:class="iconClass"
+				v-text="iconText"
+				@click.stop="$KtNavbar.toggle()"
+			/>
+		</div>
+		<div v-else :class="objectClass" @click="$emit('logoClick')">
+			<div
+				:style="
+					`background-image: url(${themeColor.logoUrl}); height: ${
+						themeColor.logoHeight
+					}`
+				"
+				v-if="themeColor.logoUrl"
```

Comment:
`v-if` should be the first thing, no the second. It’s way more important than some css.